k8s/subquery/templates/node/node.cmap.yml

```diff
@@ -8,6 +8,8 @@ metadata:
 
 data:
   DB_DATABASE: {{ .Values.db.name | quote }}
+  SHADOW_DATABASE: {{ .Values.db.shadowDatabase | quote }}
+  ROOT_DATABASE: {{ .Values.db.rootDatabase | quote 
```

Comment:
We actually shouldn't need these in production. These are only needed for resetting the DB or committing migrations. We should only ever be applying migrations in production which only interacts with `DB_DATABASE`.